ilc/client/TransactionManager.js

```diff
@@ -11,18 +11,18 @@ export const slotWillBe = {
 };
 
 export class TransactionManager {
-    #registrySettings;
+    #spinnerConfig;
     #globalSpinner;
     #spinnerTimeout;
 
     #fakeSlots = [];
     #hiddenSlots = [];
     #transactionBlockers = [];
 
-    constructor(registrySettings = {}) {
-        this.#registrySettings = Object.assign({
-            globalSpinner: true
-        }, registrySettings);
+    constructor(registrySettings = {globalSpinner: {enabled: true, customHTML: ''}}) {
```

Comment:
I am not sure that we should provide all settings here

Reply:
fixed